app/assets/stylesheets/lead_providers/statements.scss

```diff
@@ -142,10 +142,6 @@
     display: grid;
     gap: 2rem;
     grid-template-columns: 1fr;
-
-    @media (min-width: 40.625em) {
-      grid-template-columns: auto 1fr;
-    }
   }
```

Comment:
Removing this breaks the old version:
https://cpd-ecf-review-5578-web.test.teacherservices.cloud/finance/ecf/payment_breakdowns/3e304ac7-12a8-4a62-a0e4-fc41fc7cf284/statements/dc569d28-2dc5-404f-b837-4c33f6782b65

Keeping it shouldn't interfere with the new version